lib/spack/spack/test/oci/integration_test.py

```diff
@@ -25,6 +27,8 @@
 env = SpackCommand("env")
 install = SpackCommand("install")
 
+pytestmark = pytest.mark.not_on_windows("file not available on Windows")
```

Comment:
Were these tests running and passing before? I don't see any prior skips, but they use the buildcache, which _should_ be failing on a lack of Gpg/File...

FWIW I think this change is correct, I'm just confused how this was ever passing CI.

Reply:
I just ran the unit-test again and without this pytest mark all the tests fail. Originally, there was a pytest mark that just said it did not run on Windows

Reply:
Oh huh, that wasn't showing up in the diff. Thanks.